routes.py

```diff
@@ -128,7 +128,7 @@ def yield_file(chunk=8192):
     rv.headers['Accept-Ranges'] = 'bytes'
     attachment_filename = os.path.basename(path)
     rv.headers.add('Content-Disposition', 'attachment',
-                   filename=attachment_filename.encode('utf-8', 'replace').decode('utf8'))
+                   filename=attachment_filename.encode('latin-1', 'replace').decode('latin-1'))
```

Comment:
Because of this: https://github.com/python/cpython/blob/3.6/Lib/http/server.py#L508 :/